app/pages/lab/visibility.cjsx

```diff
@@ -169,6 +169,9 @@ module.exports = React.createClass
             {@state.workflows.map (workflow) =>
               setting = workflow.active
               stats_completeness_type = workflow.configuration.stats_completeness_type ? 'retirement'
+              stats_visible = !workflow.configuration.stats_hidden
+              if !workflow.active
+                stats_visible = !(workflow.configuration.stats_hidden == undefined) && stats_visible
```

Comment:
If `stats_hidden` is not set then this line becomes `stats_visible=true` because of line 172, no? `!undefined` is true in JavaScript. That's what's confusing me.